cli/generate/src/render.rs

```diff
@@ -694,6 +703,73 @@ impl Generator {
         add_line!(self, "");
     }
 
+    fn add_supertype_map(&mut self) {
+        add_line!(
+            self,
+            "static const TSSymbol ts_supertypes[SUPERTYPE_COUNT] = {{"
+        );
+        indent!(self);
+        for supertype in self.supertype_map.keys() {
+            add_line!(self, "{},", supertype);
```

Comment:
Really pikcy nit, but I think `supertype` can be moved inside the braces to match the style in the in the rest of this function.

Reply:
Thanks!